src/net/RTCP/RTCPCompoundPacket.cs

```diff
@@ -123,6 +133,27 @@ public RTCPCompoundPacket(byte[] packet)
             }
         }
 
+        public static string HexDump(byte[] bytes)

```

Comment:
It would be better if this method was not added here. Either using, and enchancing if 16 bytes lines are really required. the [existing method](https://github.com/sipsorcery-org/sipsorcery/blob/8f51ce49c4bb55b2872e54bcd4de534547664eda/src/sys/TypeExtensions.cs#L124) is preferred.